### DISCUSSION TITLE
GPS speed vs accelerometer/gyro

### DISCUSSION THREAD

**Discussion by tomttr1** [gopro/labs#808] [Q&A]
I use my GoPro 11 to record my kitesurfing sessions. 

The recording settings I use with the GPS are the following:
Start speed: 23kmh
End speed: 20kmh
Hold: 3 seconds
Repeat On
Video quality is 1080p, 120fps, linear+horizon lock and EIS auto boost.

It works quite well but the battery drains after  45 minutes, and I would like to know if and how I can use the accelerometer/gyro combo instead of the GPS to achieve the same result while prolonging battery life.
My goal is to make 1 clip per jump without the boring parts when I'm freeriding or struggling in choppy water.

Bonus question: is there any way to completely shutdown the GoPro and wake it up with sensors like gps/accel/gyro?

**Comment by dnewman-gpsw** [comment]
That seems a rather fast battery drain, not typical, although I don't do much testing at 120fps.  I don't think GPS is the reason for the short run time, so using accl/gyro wouldn't solve that issue (and wouldn't be as accurate.)  The script should have extended the runtime, as the camera is only recording the fast segments. 

Unfortunately when the camera is off, so are the sensors.  Only way to do that is used an external USB power trigger, messy and not waterproof.

**Comment by tomttr1** [comment]
Thank you for your answer, I'll try with 60fps instead of 120 maybe that'll help.
Now if anyone has experience with gyro/accel sensors while skiing, biking, surfing etc... please I'm curious to know the settings you are using
